src/components/DashboardsBar/NavigationMenu/NavigationMenuDropdownButton.js

```diff
@@ -0,0 +1,28 @@
+import i18n from '@dhis2/d2-i18n'
```

Comment:
I was a little confused about what this component was. I guess because I don't think of it as "navigating" to a dashboard. I think of this component more as a "DashboardSelector"

Reply:
But you change the route by pushing to the `history`...
> It exposes useful methods and properties that let you **navigate** back and forth through the user's history

_(Source: [MDN docs on the History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API), emphasis mine)_

I'm not really convinced that "selection" is more appropriate

Reply:
Fair point. But it's still a little misleading because there are other navigation elements. The "Edit" button, "New" button and "Print" menus. Maybe it could be called DashboardNavigator or DashboardNavigationMenuButton or similar. (Hard for me to say given I'm not a fan of really long names 😅 )